### DISCUSSION TITLE
Which compositor/display manager do the export templates target? (X11 vs KSM/DRM)

### DISCUSSION THREAD

**Discussion by rcoppy** [hiulit/Unofficial-Godot-Engine-Raspberry-Pi#12] [Q&A] [answered]
Hello, very cool to see all your progress!

Slightly silly question--when you build for the Pi using these templates do the resulting games target X11 or KSM/DRM? Trying to weigh using this versus somewhat less pi-specific (but maybe more customizable) [FRT](https://github.com/efornara/frt).

Thank you 😄

**Comment by hiulit** [comment, marked as answer]
Hey, wow! Sorry for the super late reply! 🙏 
It exports for X11. If you need KSM/DRM you'll have to use FRT.

**Reply by rcoppy** [reply]
No worries you're fine! Thanks for getting back to me 🙂
